### DISCUSSION TITLE
dock in safe mode after update

### DISCUSSION THREAD

**Discussion by yumminova** [linuxdeepin/developer-center#3454] [General | 常规讨论]
### SUMMARY | 问题概要

after update to 20.7 my dock is in safe mode, and the icons on the right are not showing up anymore. Switching for modern to efficient did nnot help.

I am an end user and dont really know what to include here, but im good in following guidance ))

### STEPS TO REPRODUCE | 复现步骤

I dont know, i just updated

### OBSERVED RESULT | 观察到的结果

dock is in safe mode

### EXPECTED RESULT | 期望的结果

dock not in safe mode

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

version 20.7

**Comment by BLumia** [comment]
Are you using deepin or other distro? Are you upgrading via the Control Center or from terminal via `apt`?

**Comment by yumminova** [comment]
I'm using deepin. Sometimes when I get notification I upgrade via control center, sometimes via apt

Verzonden met Proton Mail mobiel

-------- Oorspronkelijk bericht --------
Op 4 okt. 2022 07:08, schreef Gary Wang :

**Comment by BLumia** [comment]
Which is the one that caused the safe-mode dock issue? If it's the `apt` method, which exactly is the command you're using? `apt upgrade` or `apt dist-upgrade`?

**Comment by yumminova** [comment]
i am not sure, my brother created a script for updating:
alias updater="sudo apt-get update && sudo apt-get -y upgrade && sudo apt-get -y autoremove && sudo apt-get -y clean"

so i think its via that method

------- Original Message -------

**Comment by BLumia** [comment]
> `sudo apt-get -y upgrade`

`upgrade` and `dist-upgrade` is not the same. For commandline user, the later one is usually recommended. I guess maybe that caused the issue. Update via the Control Center is always the recommended way to update your system...

I'll convert this issue to a discussion thread since it's not really a bug. To resolve the issue, you may check the log of `apt` and see if any unintended package removal, or choose a simpler way: backup your files and reinstall the system.
